.github/workflows/ci.yml

```diff
@@ -1047,7 +1047,7 @@ jobs:
 
       - uses: Swatinem/rust-cache@v2
       - name: test tokio
-        run: wasm-pack test --node -- --features "macros sync"
+        run: wasm-pack test --node -- --features "macros sync time rt"
```

Comment:
Any chance you could expand this job to run both with and without time?

Reply:
sorry for late, I've updated how it tests